crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -1234,6 +1245,24 @@ mod tests {
             join_paths("/playground/öut path/rollup/entrypoint1.js", "~/0.js.map"),
             "/0.js.map"
         );
+
+        // path with a leading dot
+        assert_eq!(
+            join_paths(
+                "app:///_next/static/chunks/pages/_app-569c402ef19f6d7b.js.map",
+                "./node_modules/@sentry/browser/esm/integrations/trycatch.js"
+            ),
+            "app:///_next/static/chunks/pages/node_modules/@sentry/browser/esm/integrations/trycatch.js"
+        );
+
+        // webpack with only a single slash
```

Comment:
Is this really possible? `webpack://` is a custom protocol, so it should always be 2 slashes :|

Reply:
Well, there were counter examples in the A/B tests :-(